### PR TITLE
Add extra toggles and information for xp globes

### PR DESCRIPTION
**Pull request by Emrage**
- Adds toggles for all the information in xp globes
- Adds xp/hr and actions information in xp globes

Before:

After:

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java [hunk 1]
```diff
@@ -47,11 +47,77 @@ default boolean enableTooltips()
 		return true;
 	}
 
+	@ConfigItem(
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java]: All these new toggles are completely unnecessary

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobe.java [hunk 2]
```diff
@@ -123,4 +127,25 @@ public void setTime(Instant time)
 	{
 		this.time = time;
 	}
+
+	public double getXpPerHour()
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobe.java]: Use Lombok `@Getter` and `@Setter` annotations

File: runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerService.java [hunk 3]
```diff
@@ -41,4 +41,8 @@
 	 * @return
 	 */
 	int getActionsHr(Skill skill);
+
+	int getXpPerHour(Skill skill);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerService.java]: Probably add documentation for these new methods too.

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobe.java (hunk 2 above)
Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobe.java]: Also, XP/hour is imo completely out of scope of the informations that XP globes should provide, for that, there is XP tracker.

Comment (Emrage) [reply runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobe.java]: Should XP/hour be disable on default then?

**Comment by devLotto** [comment]
Actions left was added in #1636 , so what this PR adds is xp/hr and toggles.
The developer seems to have given up though.

**Comment by deathbeam** [comment]
Closing as this is very old PR now and has a lot of merge conflicts, reopen if you want to re-implement it.

**Pull request closed without merging**